apps/src/javalab/Playground.js

```diff
@@ -146,6 +190,8 @@ export default class Playground {
   reset() {
     this.isGameOver = false;
     this.isGameRunning = false;
+    // reset playground items to be empty
+    this.setPlaygroundItems({});
```

Comment:
Need to clear `this.imageItemIds` here too right?

Reply:
I'm removing `imageItemIds` to avoid this problem :)